core/client.go

```diff
@@ -0,0 +1,51 @@
+package core
+
+import (
+	"fmt"
+
+	"github.com/celestiaorg/celestia-core/config"
+	corenode "github.com/celestiaorg/celestia-core/node"
+	"github.com/celestiaorg/celestia-core/rpc/client"
+	"github.com/celestiaorg/celestia-core/rpc/client/http"
+	"github.com/celestiaorg/celestia-core/rpc/client/local"
+)
+
+// Client is an alias to Core Client.
+type Client = client.Client
+
+// NewRemote creates a new Client that talks to remote Core over HTTP.
+func NewRemote(protocol, remoteAddr string) (Client, error) {
+	return http.New(
+		fmt.Sprintf("%s://%s", protocol, remoteAddr),
+		"/websocket",
+	)
+}
+
+// NewEmbedded creates a new Client which works over embedded Core in the same proc.
```

Comment:
```suggestion
// NewEmbedded creates a new embedded Core node process. 
```

Reply:
That's not correct.

Reply:
It creates a Client first of all

Reply:
What exactly does `corenode.DefaultNewNode` do then? This creates an actual core node and wraps it. `Node` implements the `Client` behaviour.

Reply:
It creates an embedded core.Client first of all. The fact that it uses a node underneath is an implementation detail. The constructor returns Client, not a Node, thus we should say it returns Client, not a Node. Instead, we can say something like "creates a new Client implemented over a core Node running in the same process." I am happy to get a similar suggestion, but this one is not correct.

Reply:
```suggestion
// NewEmbedded returns a new Client from an embedded Core node process.
```